.travis.yml

```diff
@@ -16,14 +17,15 @@ matrix:
   fast_finish: true
 
 before_install:
-  - npm config set spin false
-  - npm install -g bower # bower needs to be installed globally for ember-cli-addon-tests
-  - npm install -g npm # make sure npm is updated for ember-cli
-  - npm install phantomjs-prebuilt
+  # Since we override the install hooks, we need to setup Yarn on our own
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+  # NPM & Bower need to be installed globally for ember-cli-addon-tests
+  - yarn global add npm bower phantomjs-prebuilt
```

Comment:
It's still needed for ember-cli-addon-tests. Since `yarn` isn't supported yet, it falls back to using `npm`.

See: https://github.com/tomdale/ember-cli-addon-tests/issues/76